browser/ui/brave_browser_command_controller.cc

```diff
@@ -206,8 +209,8 @@ void BraveBrowserCommandController::UpdateCommandForBraveVPN() {
   UpdateCommandEnabled(IDC_ABOUT_BRAVE_VPN, true);
   UpdateCommandEnabled(IDC_MANAGE_BRAVE_VPN_PLAN, true);
 
-  if (auto* vpn_service =
-          BraveVpnServiceFactory::GetForProfile(browser_->profile())) {
+  if (auto* vpn_service = brave_vpn::BraveVpnServiceFactory::GetForProfile(
```

Comment:
Should we use a raw_ptr<...> here?

Reply:
That makes sense. Thanks!

The difference seems to be that the address is zeroed on definition, which may work even for non-member classes as a security mechanism. Indeed it's almost always exploited in classes initialization cases.